ftest/web-ui/webdriver/test/features/publication.feature

```diff
@@ -47,6 +51,7 @@ Feature: Internal Publication
 
   Scenario: Republish Current Document
     Given  I have a File document
+    And I login as "Administrator"
```

Comment:
For readability purposes it would more clear "Given I login ... And I have a File.."